dpdispatcher/openapi_context.py

```diff
@@ -0,0 +1,252 @@
+import os
+import shutil
+import uuid
+
+import tqdm
+from bohriumsdk.client import Client
+from bohriumsdk.job import Job
+from bohriumsdk.storage import Storage
+from bohriumsdk.util import Util
+
+from dpdispatcher import dlog
+from dpdispatcher.base_context import BaseContext
+from dpdispatcher.JobStatus import JobStatus
+
+# 引入一个sqlite数据库，存储job相关信息
+DP_CLOUD_SERVER_HOME_DIR = os.path.join(
+    os.path.expanduser("~"), ".dpdispatcher/", "dp_cloud_server/"
+)
+
+
+class OpenAPIContext(BaseContext):
```

Comment:
Documentation should be added for a new context [here](https://github.com/deepmodeling/dpdispatcher/blob/8ee05a704815745bfb32a91493a772df9d29b6a6/doc/context.md).